tests/data/config/run.config

```diff
@@ -0,0 +1,14 @@
+ESTUARY_TOKEN=ESTabcdefghARY
```

Comment:
This is not going to pass as a valid estuary token (has to have a valid uuid between `EST` and `ARY`)